### PR TITLE
feature/i18n

### PR DESCRIPTION
**Pull request by maxence-lefebvre**

### CODE REVIEW

File: src/react/AppWrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import PropTypes from 'prop-types';
+import { IntlProvider } from 'react-intl';
+
+import messages from 'i18n/messages.json';
+
+// @todo maxence-lefebvre
+// I should get react-intl locale too...
+// But first, I'll connect redux to the application
+
+const AppWrapper = props => (
```
Comment (gautier-lefebvre) [inline src/react/AppWrapper.jsx]: don't forget to set a displayName

File: config/polyfills.js [hunk 2]
```diff
@@ -20,3 +20,7 @@ Object.assign = require('object-assign');
 if (process.env.NODE_ENV === 'test') {
   require('raf').polyfill(global);
 }
+
+// Needed by async/await and Babel 7
+require("babel-polyfill");
```
Comment (gautier-lefebvre) [inline config/polyfills.js]: shouldn't it be `@babel/polyfill` ? this only works thanks to the babel-core@7.0.0-bridge

File: src/services/i18n/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+/**
+ * Stores locale in the storage.
+ *
+ * @param {string} locale - The locale to store.
+ */
+export const storeLocale = (locale) => {
+  localStorage.setItem(LOCALE_STORAGE_KEY, locale);
+};
+
+export const setHtmlLangAttribute = (locale) => {
```
Comment (gautier-lefebvre) [inline src/services/i18n/index.js:44]: `document?.getElementsByTagName('html')[0]?.setAttribute('lang', locale);`

Comment (maxence-lefebvre) [reply src/services/i18n/index.js:44]: Did you activate optional chaining babel plugin ?

Comment (gautier-lefebvre) [reply src/services/i18n/index.js:44]: It's in the stage-1 preset, so yes.

File: package.json [hunk 4]
```diff
@@ -102,6 +108,8 @@
     "postcss-loader": "^2.0.8",
     "raf": "^3.4.0",
     "react-dev-utils": "^5.0.0",
+    "react-intl-cra": "^0.3.3",
```
Comment (sylvainar) [inline package.json:111]: Do we actually need this ?

Comment (maxence-lefebvre) [reply package.json:111]: Yes, it is the module that extracts the messages from `<FormattedMessage />` `<FormattedHTMLMessage />`, `defineMessage()`

File: src/services/i18n/index.js (hunk 3 above)
Comment (maxence-lefebvre) [reply src/services/i18n/index.js:44]: Eslint does not agree with you

**Review by gautier-lefebvre — APPROVED** [review]

**Pull request merged**